doc/src/base/collections.md

```diff
@@ -200,7 +200,6 @@ Base.WeakKeyDict
 Base.ImmutableDict
 Base.haskey
 Base.get(::Any, ::Any, ::Any)
```

Comment:
This line should be removed instead, since there is also documentation for `get(::Function, collection, key)` and both can be grouped under `Base.get`.